src/msg.c

```diff
@@ -696,10 +698,6 @@ static ssize_t msg_process(struct session *session,
 		if (session->schema_timeout)
 			l_timeout_remove(session->schema_timeout);
```

Comment:
This snippet of code shouldn't be replaced in another place since the `session->schema_timeout` will be allocated in another function?